templates/endpoints_v2/README.ipynb

```diff
@@ -18,22 +18,24 @@
     "\n",
     "## Step 1 - Run the model locally in the Workspace\n",
     "\n",
-    "The llm-serve.yaml file in this example runs the Mistral-7B model. There are 2 important configurations you would need to modify:\n",
-    "1. The `models` config in `llm-serve-*.yaml` contains a list of YAML files for the models you want to deploy. You can run any of the models in the `models` directory or define your own model YAML file and run that instead. All config files follow the naming convention `{model_name}_{accelerator_type}_{tensor_parallelism}`. Follow the CustomModels [guide](examples/CustomModels.ipynb) for bringing your own models.\n",
-    "2. `HUGGING_FACE_HUB_TOKEN` - The Meta Llama-2 family of models need the HUGGING_FACE_HUB_TOKEN variable to be set to a Hugging Face Access Token for an account with permissions to download the model.\n",
-    "\n"
+    "The following command will allow you to serve a model based on:\n",
+    "- Model ID\n",
+    "- GPU type\n",
+    "- Tensor parallelism\n",
+    "\n",
+    "The script creates a YAML file and serves the model via ray serve. You can also run any of the models in the `models` directory or define your own model YAML file and run that instead. You can follow the CustomModels [guide](examples/CustomModels.ipynb) for bringing your own models.\n",
+    "\n",
+    "\n",
+    "Please note that the Meta Llama-2 family of models need the HUGGING_FACE_HUB_TOKEN variable to be set to a Hugging Face Access Token for an account with permissions to download the model.\n"
    ]
   },
   {
-   "cell_type": "markdown",
+   "cell_type": "code",
+   "execution_count": null,
    "metadata": {},
+   "outputs": [],
    "source": [
-    "From the VSCode terminal (press [**Ctrl + `**] in VSCode), use the Ray Serve CLI to deploy the model for testing. It will take a few minutes to initialize and download the model.\n",
-    "\n",
-    "```bash\n",
-    "# Note: if using GCP cloud, use llm-serve-gcp.yaml instead to select L4 GPU instances.\n",
-    "$ serve run llm-serve-aws.yaml\n",
-    "```"
+    "!python main.py"
```

Comment:
So the Ray Serve processes started by it won't block the following cells' execution I guess?